### PR TITLE
testing/gdal: upgrade to 3.0.0

### PR DESCRIPTION
**Pull request by abellgithub**
Works with proj version 6.

### CODE REVIEW

**Comment by maxice8** [comment]
use our commit title scheme `<repo>/<pkgname>: upgrade to <version>`

you can add why on the body of the commit

File: testing/gdal/APKBUILD [hunk 1]
```diff
@@ -23,6 +23,7 @@ makedepends="
 	swig
 	tiff-dev
 	zlib-dev
+    proj-dev
```
Comment (maxice8) [inline testing/gdal/APKBUILD]: identation is wrong

**Comment by maxice8** [comment]
Please follow our commit naming scheme, it **will not** be merged otherwise

**Comment by abellgithub** [comment]
A link to requirements would be helpful.

**Pull request closed without merging**